Api/Controllers/NamesController.cs

```diff
@@ -164,5 +165,29 @@ public async Task<IActionResult> DeleteName(string name)
             await _nameEntryService.Delete(name);
             return Ok(new { Message = $"{name} Deleted" });
         }
+
+
+        [HttpDelete("batch")]
+        public async Task<IActionResult> DeleteNamesBatch(string[]names)
+        {
+
+            var foundNames = (await _nameEntryService.LoadNames(names))?.Select(f=>f.Name)?.ToArray();
```

Comment:
Not sure, but I think this method would either return an empty array (not a null) if no names are found. Please test and modify the conditional statement below if necessary.

Reply:
Handled empty array.